src/browsers/schema/content/about.md

```diff
@@ -0,0 +1,46 @@
+
+TWe would also like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Foundation](https://www.broadinstitute.org/stanley), [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), Kent and Elizabeth Dauten, and The Dalio Foundation, and the leadership of its members.
+
+# Contributing scientists
+
+Tarjinder  Singh^1,2^, Timothy  Poterba^1,2^, David  Curtis^3,4^, Huda  Akil^5^, Mariam  Al Eissa^6^, Jack D. Barchas^5^, Nicholas  Bass^6^, Tim B. Bigdeli^7^, Gerome  Breen^8^, Evelyn J. Bromet^9^, Peter F. Buckley^10^, William E. Bunney^11,12^, Jonas  Bybjerg-Grauholm^13^, William F. Byerley^5,14^, Sinéad B. Chapman^2^, Wei J. Chen^15^, Claire  Churchhouse^1,2^, Nicholas  Craddock^16^, Charles  Curtis^8^, Caroline M. Cusick^2^, Lynn  DeLisi^5^, Sheila  Dodge^17^, Michael A. Escamilla^18^, Saana  Eskelinen^19,20^, Ayman H. Fanous^21^, Stephen V. Faraone^22^, Alessia  Fiorentino^6^, Laurent  Francioli^1,23^, Stacey B. Gabriel^17^, Diane  Gage^2^, Sarah A. Gagliano Taliun^24^, Andrea  Ganna^1,25^, Giulio  Genovese^2^, David C. Glahn^26^, Jakob  Grove^27^, Mei-Hua  Hall^28^, Eija  Hämäläinen^25^, Henrike O. Heyne^1,2,29^, Matti  Holi^30^, David M. Hougaard^13^, Daniel P. Howrigan^1,2^, Hailiang  Huang^1,2^, Hai-Gwo  Hwu^31^, René S. Kahn^32,33^, Hyun Min  Kang^24^, Konrad J. Karczewski^1,2^, George  Kirov^34^, James A. Knowles^35^, Francis S. Lee^11,36^, Douglas S. Lehrer^37^, Francesco  Lescai^38,39^, Dolores  Malaspina^32^, Stephen R. Marder^32^, Steven A. McCarroll^2,40^, Helena  Medeiros^21^, Lili  Milani^41^, Christopher P. Morley^42^, Derek W. Morris^43^, Preben Bo  Mortensen^44^, Richard M. Myers^11,45^, Merete  Nordentoft^46,47^, Niamh L. O'Brien^6^, Ana Maria  Olivares^2^, Dost  Ongur^28^, Willem H. Ouwehand^48^, Duncan S. Palmer^1,2^, Tiina  Paunio^49^, Digby  Quested^50^, Mark H. Rapaport^51^, Elliott  Rees^34^, Brandi  Rollins^11,12^, F. Kyle  Satterstrom^2,52^, Alan  Schatzberg^5^, Edward  Scolnick^2^, Laura J. Scott^24^, Sally I. Sharp^6^, Pamela  Sklar, Jordan W. Smoller^53,54^, Janet l. Sobell^55^, Matthew  Solomonson^23^, Christine R. Stevens^2,23^, Jaana  Suvisaari^56^, Grace  Tiao^23^, Stanley J. Watson^57^, Nicholas A. Watts^23^, Douglas H  Blackwood^58^, Anders D. Børglum^59,60^, Bruce M. Cohen^28^, Aiden P. Corvin^61^, Tõnu  Esko^41^, Nelson B. Freimer^62^, Stephen J. Glatt^22^, Christina M. Hultman^63^, Andrew  McQuillin^6^, Aarno  Palotie^23,25^, Carlos N. Pato^7^, Michele T. Pato^7^, Ann E. Pulver^64^, David  St. Clair^65^, Ming T. Tsuang^66^, Marquis P. Vawter^11,67^, James T. Walters^34^, Thomas M. Werge^68,69,70,71^, Roel A. Ophoff^62,72^, Patrick F. Sullivan^73,74^, Michael J. Owen^34^, Michael  Boehnke^24^, Michael C. O'Donovan^34^, Benjamin M. Neale^1,2,23^, Mark J. Daly^1,2,23,25^
```

Comment:
Not sure if our Markdown parser supports `^` for superscript. Might have to use HTML (`<sup>...</sup>`) for these.

Reply:
Changed it to <sup>